style.css

```diff
@@ -0,0 +1,107 @@
+* {
+    padding: 0;
+    margin: 0;
+  }
+  
+  .title-container {
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 15px;
+  }
+  
+  .title-container button img {
```

Comment:
Please Use **.title-container > button > img** instead of  **.title-container button img**; 
The One You're using will Select Every Button; everywhere inside **title-container** class;
And Every Image; Everywhere Inside the Button.